controllers/host/host_controller.go

```diff
@@ -2235,6 +2238,7 @@ func (r *HostReconciler) Reconcile(ctx context.Context, request ctrl.Request) (r
 	if instance.Status.ObservedGeneration == instance.ObjectMeta.Generation &&
 		instance.Status.Reconciled &&
 		instance.Status.DeploymentScope == "bootstrap" &&
+		instance.Status.StrategyRequired == cloudManager.StrategyNotRequired &&
```

Comment:
Same comment here: if the scope is "bootstrap", why do we want to check the strategy again?

Reply:
Same answer applies here as well:

> No, in case of network reconfiguration, scope is not changed for host, it will only be updated for platformnetwork so it'll not help.